native/src/main/java/io/ballerina/stdlib/protobuf/deserializers/Fixed32Deserializer.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.protobuf.deserializers;
+
+import com.google.protobuf.Descriptors;
+import io.ballerina.runtime.api.types.Type;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BArray;
+import io.ballerina.runtime.api.values.BMap;
+import io.ballerina.runtime.api.values.BString;
+import io.ballerina.stdlib.protobuf.messages.BMessage;
+
+import java.io.IOException;
+
+/**
+ * The deserializer class, that deserializes the fixed 32 data.
+ */
+@SuppressWarnings("unchecked")
+public class Fixed32Deserializer extends AbstractDeserializer {
+
+    public Fixed32Deserializer(com.google.protobuf.CodedInputStream input, Descriptors.FieldDescriptor fieldDescriptor,
+                               BMessage bMessage, Type targetType, boolean isPacked) {
+
+        super(input, fieldDescriptor, bMessage, targetType, isPacked);
+    }
+
+    @Override
+    public void deserialize() throws IOException {
+
+        BString bFieldName = StringUtils.fromString(fieldDescriptor.getName());
+        if (isBMap()) {
+            BMap<BString, Object> bMap = (BMap<BString, Object>) bMessage.getContent();
+            if (fieldDescriptor.isRepeated()) {
+                BArray int32Array = (BArray) bMap.get(bFieldName);
```

Comment:
```suggestion
                BArray fixed32Array = (BArray) bMap.get(bFieldName);
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-protobuf/pull/176/commits/0b4e304a70313de3a78afb5bc4ffe5ce99f8cd6b